docs/use-cases/tokenization/real-world-assets.page.tsx

```diff
@@ -1,9 +1,10 @@
 import { useThemeHooks } from "@redocly/theme/core/hooks";
 import { Link } from "@redocly/theme/components/Link/Link";
+import { useEffect } from "react";
 
 export const frontmatter = {
   seo: {
-    title: "Real-World Asset Tokenization",
+    title: "RWA Tokenization on XRP Ledger Blockchain",
```

Comment:
Per the discussion in Slack, let's leave the title unchanged ( for the same reasons noted in https://github.com/XRPLF/xrpl-dev-portal/pull/2848#discussion_r1849135787 )